src/stylesheets/common/_colors.scss

```diff
@@ -57,46 +57,75 @@ $okColor:#63a240;
 
 
 //Classes for styleguide
```

Comment:
For background colors, how do you feel about calling these `bg-[color]` or `background-[color]` instead of `mz-[color]`?  We could also have a set of classes to control text color (probably best not to combine those with background color anyway). Something like  `type-reversed` for when you need `color: #fff` (or whatever color you'd want over a dark background).

Also, are you OK with all of these being used as background colors or would you rather limit to a more controlled palette?  (In which case, we could move towards a more semantic classification, like `bg-primary` or `bg-lowlight`, etc.)